### PR TITLE
plots.test.ts: update ipytree test to assert expanding/collapsing of nodes

### PR DESCRIPTION
**Pull request by midleman**
### Intent

The current ipytree plot test simply verifies the plot renders. It seems like better assertions around behavior would be beneficial, in particular the main functionality of expanding/collapsing nodes.

### Approach
* Updated the shared simplePlotTest function to support additional assertions after the plot has rendered.
* Added assertions to verify:
   * The expected number of nodes are visible after the plot initially renders.
   * The tree behaves as expected when collapsing the parent node, by asserting the number of visible nodes decreases accordingly.

### QA Notes

Confirmed locally that the entire plot test suite (`plots.tests.ts`) passed without any issues.

### CODE REVIEW

**Comment by midleman** [comment]
I have read the CLA Document and I hereby sign the CLA.

File: test/smoke/src/areas/positron/plots/plots.test.ts [hunk 1]
```diff
@@ -398,6 +395,13 @@ tree`;
 
 				await simplePlotTest(app, script, '.jstree-container-ul');
 
+				// tree should be expanded by default
+				const treeNodes = app.workbench.positronPlots.getWebviewPlotLocator('.jstree-container-ul .jstree-node');
+				await expect(treeNodes).toHaveCount(9);
+
+				// collapse the tree, only parent nodes should be visible
+				treeNodes.first().click({ position: { x: 0, y: 0 } }); // target the + icon
+				await expect(treeNodes).toHaveCount(3);
```
Comment (jonvanausdeln) [inline test/smoke/src/areas/positron/plots/plots.test.ts:404]: Not sure why.. but on my box (Windows 11), something about this causes the next test to fail.  I verified this by running the tests on `main` and they pass without issue.

File: test/smoke/src/areas/positron/plots/plots.test.ts [hunk 2]
```diff
@@ -414,7 +418,7 @@ output`;
 
 				// Redirect a print statement to the Output widget.
 				await app.workbench.positronConsole.pasteCodeToConsole(`with output:
-    print('Hello, world!')
+print('Hello, world!')
```
Comment (jonvanausdeln) [inline test/smoke/src/areas/positron/plots/plots.test.ts:421]: This is the change that breaks for me.   Without the indentation, I get
```
>>> with output:
... print('Hello, world!')
IndentationError: expected an indented block after 'with' statement on line 1 (4154732719.py, line 2)
```
Just curious why it was removed?

Comment (midleman) [reply test/smoke/src/areas/positron/plots/plots.test.ts:421]: Oh weird. I got an error on the husky pre-commit hook about the white space here. 
> test/smoke/src/areas/positron/plots/plots.test.ts(421,1): Bad whitespace indentation

Which is why I edited it and then it was happy. Soooo? I can put it back but I'll just have to pass in `--no-verify` to skip the checks... First tho, I'll try reverting that change in the morning and confirm I wasn't imagining the precommit failure.

File: test/smoke/src/areas/positron/plots/plots.test.ts (hunk 1 above)
Comment (midleman) [reply test/smoke/src/areas/positron/plots/plots.test.ts:404]: Oh weird, so it's just _your_ machine, but not CICD land. Usually it's the opposite. 😂

Comment (jonvanausdeln) [reply test/smoke/src/areas/positron/plots/plots.test.ts:404]: Ha! It's probably a windows issue... something about how python on windows interprets whitespace/tabs, etc.

**Review by jonvanausdeln — APPROVED** [review]
LGTM! Works on my machine now!

**Pull request merged**